CNAME

```diff
@@ -1 +0,0 @@
-syt.hebron.wtf
```

Comment:
We don't need this?

Reply:
Oh yeah it's a part of the CI now that's right

Reply:
its actually defined already in https://github.com/happy-software/shuffle_youtube_playlist/blob/gh-pages/CNAME